File: Selenium/Activities/Activity1.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;


public class Activity1 {
    public static void main(String[] args) {

        // Installing Firefox driver
        WebDriverManager.firefoxdriver().setup();

        // Suppressing browser log
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "NULL");
        // Initialize firefox driver
        WebDriver driver = new FirefoxDriver();

        // Open a page
        driver.get("https://www.training-support.net");

        System.out.println("Page Title is: " + driver.getTitle());

        WebElement aboutUs = driver.findElement(By.id("about-link"));
        aboutUs.click();
        System.out.println("Next Page Title is: " + driver.getTitle());

        driver.quit();

    }
}

File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.common.by import By

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service = service) as driver:

    # Open the link in browser
    driver.get("https://www.training-support.net")

    print("Page Title is: " + driver.title)

    aboutUs = driver.find_element(By.ID, "about-link")
    aboutUs.click()

    print("Next Page Title is: " + driver.title)

    driver.quit()